pkg/resource/private_endpoint_registration.go

```diff
@@ -126,19 +126,15 @@ func (r *PrivateEndpointRegistrationResource) Read(ctx context.Context, req reso
 		}
 	}
 
-	if privateEndpoint == nil {
-		resp.Diagnostics.AddError("Private endpoint not found", "Could not find private endpoint in org registration")
```

Comment:
Maybe include private endpoint Name here so the user knows which one?

Reply:
this code was removed.